src/main/java/com/minecolonies/core/client/gui/modules/WindowListRecipes.java

```diff
@@ -184,12 +184,18 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
                 List<ItemStack> displayStacks = recipe.getRecipeType().getOutputDisplayStacks();
                 icon.setItem(displayStacks.get((lifeCount / LIFE_COUNT_DIVIDER) % (displayStacks.size())));
 
-                if (!module.isRecipeAlterationAllowed())
+                final Button removeButton = rowPane.findPaneOfTypeByID(BUTTON_REMOVE, Button.class);
+                if (removeButton != null)
                 {
-                    final Button removeButton = rowPane.findPaneOfTypeByID(BUTTON_REMOVE, Button.class);
-                    if (removeButton != null)
+                    if (module.isRecipeAlterationAllowed())
+                    {
+                        removeButton.setVisible(true);
+                        removeButton.setEnabled(recipe.getRecipeSource() == null || Screen.hasControlDown());
+                    }
+                    else
                     {
                         removeButton.setVisible(false);
+                        removeButton.setEnabled(false);
```

Comment:
```suggestion
                        removeButton.off();
```

Reply:
That's not an improvement.  This is deliberately mirroring the other if branch.